specs/waku/waku-v2.md

```diff
@@ -310,6 +314,67 @@ implementation matches spec
 
 Initial raw version. Released []()
 
+# Upgradability and Compatibility
+## Compatibility with Waku v1
+
+Waku v2 and Waku v1 are different protocols all together. They use a different
+transport protocol underneath; Waku v1 is devp2p RLPx based while Waku v2 uses
+libp2p. The protocols themselves also differ as does their data format.
+Compatibility can be achieved only by using a bridge that not only talks both
+devp2p RLPx and libp2p, but that also transfers (partially) the content of a
+packet from one version to the other.
+
+### Bridge
+
+A bridge requires supporting both Waku versions:
+
+* Waku v1 - using devp2p RLPx protocol
+* Waku v2 - using libp2p protocols
+
+Packets received on the Waku v1 network SHOULD be published just once on the
+Waku v2 network. More specifically, the bridge SHOULD publish
+this through the Waku Relay (PubSub domain).
+
+Publishing such packet will require the creation of a new `Message` with a
+new `WakuMessage` as data field. The `data` and `topic` field from the Waku v1
+`Envelope` MUST be copied to the `payload` and `contentTopic` fields of the
+`WakuMessage`. Other fields such as nonce, expiry and ttl will be dropped as
```

Comment:
Separate from this PR, but I wonder if there's something we are missing as we drop these fields

Reply:
Well, nonce is only there for PoW, which isn't there anymore.

I'm not sure how message lifetime works in pubsub. I've tried to figure it out before, but I only noticed the use of a timed cache. No information of ttl within the message format itself afaik. 
But even if there was, I doubt you could insert ttl values manually (API probably wouldn't allow that?), so you'd have to hack it in there. Probably not the best idea.